docs/user/account/data-collection.md

```diff
@@ -0,0 +1,27 @@
+# Data Collection
+
+When you use the Graasp platform we collect usage data.
+This article describes what data we collect and who we share it with.
+
+## Purpose of the data collection
+
+When you create your account you have the possibility to allow or reject the **collection of data for education purposes**.
+If you accept, we will collect data related to the way you interact and consume the learning materials on the platform.
+This data will be accessible to the material authors to allow to analyze how their content is consumed and how learners interact with it.
+The data we collect is not shared with third parties outside the platform.
+
+## Opting out of data collection after account creation
+
+You can opt out of data collection at any time in your account settings. No data will be collected when you browse learning material in the future.
+
+## Deleting your data
+
+You can request a complete data deletion by contacting us via email: [data-management@graasp.org](mailto:data-management@graasp.org?subject=Data%20Deletion).
+
+Alternatively you can [Delete your account](delete-account). This will anonymise your data and delete your traces inside our system to the notable exception of:
+
+- the elements you have created since they might be shared with other users. To ensure they are removed, you should trash them (and delete them permanently from the trash) before deleting you account.
```

Comment:
```suggestion
- the items you have created since they might be shared with other users. To ensure they are removed, you should trash them (and delete them permanently from the trash) before deleting you account.
```